Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -133,14 +138,20 @@ echo disk > /sys/power/state
 		Write-LogInfo "Rebooting All VMs!"
 		$TestProvider.RestartAllDeployments($AllVMData)
 
-		# Check the VM status before hibernation
-		$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
-		if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
-			Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is running before hibernation"
-		} else {
-			Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status before hibernation"
-			throw "Can not identify VM status before hibernate"
-		}
+		For ($iteration=1;$iteration -le $defaultHibernateLoop; $iteration++) {
+			if ($defaultHibernateLoop -ne 1) {
+				Write-LogInfo "Running Hibernation stress test in the iteration - $iteration"
+				# Clear dmesg log before running test
+				Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg -c" -runAsSudo -ignoreLinuxExitCode:$true | Out-Null
+			}
+			# Check the VM status before hibernation
+			$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
+			if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
+				Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is running before hibernation"
+			} else {
+				Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status before hibernation"
```

Comment:
"Could not find the VM status before hibernation" is not a correct description

Reply:
Fixed in ae66ef26caf39800d996c15cf19eed2b44ecfc9f

Reply:
Fixed in 389c7fdf2b96277a1b0c34ee9bbf34441c63dbec